global.json

```diff
@@ -1,16 +1,17 @@
 {
   "sdk": {
+    "version": "8.0.101",
```

Comment:
what does 2 versions mean here?

Reply:
LOL how did this pass? I guess the later wins and just overwrites the former. Sorry that was a merge conflict and I just submitted https://github.com/dotnet/aspire/pull/1629 to fix that.